File: src/main/java/com/dh/ondot/member/core/exception/InvalidTokenHeaderException.java
package com.dh.ondot.member.core.exception;

import com.dh.ondot.core.exception.UnauthorizedException;

import static com.dh.ondot.core.exception.ErrorCode.INVALID_TOKEN_HEADER;

public class InvalidTokenHeaderException extends UnauthorizedException {
    public InvalidTokenHeaderException() {
        super(INVALID_TOKEN_HEADER.getMessage());
    }

    @Override
    public String getErrorCode() {
        return INVALID_TOKEN_HEADER.name();
    }
}

File: src/main/kotlin/com/dh/ondot/member/core/exception/InvalidTokenHeaderException.kt
package com.dh.ondot.member.core.exception

import com.dh.ondot.core.exception.ErrorCode.INVALID_TOKEN_HEADER
import com.dh.ondot.core.exception.UnauthorizedException

class InvalidTokenHeaderException :
    UnauthorizedException(INVALID_TOKEN_HEADER.message) {
    override val errorCode: String get() = INVALID_TOKEN_HEADER.name
}
